changelog/unreleased/enhancement-rename-user-management

```diff
@@ -0,0 +1,5 @@
+Enhancement: Rename "user management" app
+
+The "user management" app has been renamed to "admin settings" because the app will contain a variety of general settings in the future.
+
+https://github.com/owncloud/web/pull/8175
```

Comment:
It's a breaking change, as it potentially needs a config adjustment by the user. 


We could also do a graceful deprecation and keep the old name working in config but pring a warning when it's used... 

Then it should also be added to the deprecations ticket

Reply:
Hmm I'm not sure actually, according to the Jira ticket, it's not a breaking change due to https://github.com/owncloud/web/pull/8175/files#diff-9521d586d9cc2dae8e0510f697d29d0fc9b164419ebc855a5bca4e216414e96e. That also prints a warning

Reply:
Oh you're right, apparently scrolled over that! Happy with that, nvm!

Reply:
Can you still add it to the deprecations list?

Reply:
Done so, thx for the hint!